public/css/scss/main.scss

```diff
@@ -0,0 +1,13 @@
+//fonts
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap');
+// colors
+@import "modules/colors.scss";
+// partials
+@import "partials/nav.scss";
+@import "partials/footer.scss";
+
+// animation
+@import "partials/loadingstate.scss";
+
+// pages
+@import "./modules/all.scss";
```

Comment:
I believe you don't have to add .scss at the end, nor ./ but only the / should be fine 🙂.

Reply:
can you give an example

Reply:
well line 13 would be, `@import '/modules/all'` I believe.